src/js/NativeHls.js

```diff
@@ -154,6 +154,17 @@ class NativeHls extends Meister.MediaPlugin {
         // Trigger this to make it look pretty.
         // Loading the first playlist.
         const manifest = await this.loadManifest(item.src);
+
+        if (manifest.keyInfo) {
+            const drmServerUrl = manifest.keyInfo.URI.replace('skd:', 'https:');
```

Comment:
Is URI a mandatory property of the keyInfo object? My guess is no, so we should check whether it is present before doing anything with it and sending an event.